garrysmod/gamemodes/terrortown/gamemode/cl_init.lua

```diff
@@ -368,6 +370,9 @@ function CheckIdle()
 
          timer.Simple(0.3, function()
                               RunConsoleCommand("ttt_spectator_mode", 1)
+                               net.Start("TTT_Spectate")
+                                 net.WriteBool(true)
+                               net.SendToServer()

```

Comment:
It should not be a problem if you are moved twice into spectator mode.
This shouldn't impact anything.